Cartesian_kernel/include/CGAL/Cartesian/function_objects.h

```diff
@@ -535,6 +535,40 @@ namespace CartesianKernelFunctors {
     }
   };
 
+  template <typename K>
+  class Compare_signed_distance_to_line_2
+  {
+    typedef typename K::Point_2   Point_2;
+    typedef typename K::Line_2    Line_2;
+    typedef typename K::Equal_2   Equal_2;
+  public:
+    typedef Comparison_result   result_type;
+
+    result_type
+    operator()(const Point_2& a, const Point_2& b,
+               const Point_2& c, const Point_2& d) const
+    {
+      CGAL_kernel_precondition_code(Equal_2 equal;)
+      CGAL_kernel_precondition(! equal(a,b));
+      return cmp_signed_dist_to_lineC2( a.x(), a.y(),
+					b.x(), b.y(),
+					c.x(), c.y(),
+					d.x(), d.y());
+    }
+    /*
```

Comment:
Why is this commented? If it's useless or broken, it should probably be removed.

Reply:
removed.